templates/CONTRIBUTING_GUIDE.md

```diff
@@ -0,0 +1,121 @@
+# Contributing
+
+This package is maintained by the [TEAM] team. Ask for help or a review in the [#[TEAM SLACK NAME]]([TEAM SLACK NAME LINK]) Slack channel.
+
+## :gear: Setup
+
+<!-- When applicable, list system requriements to work on the project. -->
+
+The following setup is required to work on this project:
+
+- Prerequisite 1
+- Prerequisite 2
+- Prerequisite 3
+
+## :memo: Project-specific notes
+
+<!-- Share information about the repository. -->
+<!-- What specific knowledge do contributors need? -->
+
+> [!TIP]
+> Please update this section with helpful notes for contributors.
+
+## :construction_worker: Develop
+
+> [!NOTE]
+> It's highly recommended to discuss your changes with the [TEAM] team before starting.
+>
+> A short discussion can accellerate your work and ship it faster.
+
+```sh
+# Clone and prepare the project.
+git clone [REPO SSH ADRESS]
+cd [REPO NAME]
+[DEPENDENCY INSTALL COMMAND]
+
+# Create a new branch for your changes (e.g. lh/fix-win32-paths).
+git checkout -b <your-initials>/<feature-or-fix-description>
+
+# Start the development server.
+# Run this command while you are working on your changes.
+[DEV SERVER COMMAND]
+
+# Build the project for production.
+# Run this command when you want to see the production version.
+[BUILD COMMAND]
+
+# Lint your changes before requesting a review. No errors are allowed.
+[LINT COMMAND]
+
+# Format your changes before requesting a review. No errors are allowed.
+[FORMAT COMMAND]
+
+# Test your changes before requesting a review.
+# All changes should be tested. No failing tests are allowed.
+[TEST COMMAND]
```

Comment:
I think i'd separate tests from development.
As it can be a very messy part so i'd make it separated on his own.

Reply:
That's a good idea since some repos require a specialized setup. Packages usually have very small requirements (just run `npm run test`), while services can have large requirements (run a server + configure .env + run the script).

Reply:
That's a good idea since some repos require a specialized setup. Packages usually have very small requirements (just run `npm run test`), while services can have large requirements (run a server + configure .env + run the script).